sdk/spring/spring-integration-azure-core/src/main/java/com/azure/spring/integration/core/implementation/instrumentation/InstrumentationSendCallback.java

```diff
@@ -10,7 +10,7 @@
 /**
  *
  */
-@SuppressWarnings("deprecation")
+@SuppressWarnings({"deprecation", "removal"})
 public class InstrumentationSendCallback implements ListenableFutureCallback<Void> {
```

Comment:
If we change to the recommended class, what's the minimal version of Spring Boot that having that class?

Reply:
The recommend class is from JDK,  as of 6.0, in favor of java.util.concurrent.CompletableFuture.whenComplete(BiConsumer)